packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -135,89 +179,126 @@ const AdvertiserPage = observer(props => {
-                            <div className='advertiser-page__stats-cell-completion'>
-                                <div className='advertiser-page__stats-cell-info'>
-                                    {total_completion_rate ? `${total_completion_rate}%` : '-'}
-                                </div>
-                                <div className='advertiser-page__stats-cell-info_buy'>
-                                    {localize('(Buy {{- buy_completion_rate }})', {
-                                        buy_completion_rate: buy_completion_rate
-                                            ? `${buy_completion_rate}%`
-                                            : localize('N/A'),
-                                    })}
-                                </div>
-                            </div>
-                        </Table.Cell>
-                        <div className='advertiser-page__stats-cell-separator' />
-                        <Table.Cell className='advertiser-page__stats-cell'>
-                            <div className='advertiser-page__stats-cell-header'>{localize('Avg. release time')}</div>
-                            <div className='advertiser-page__stats-cell-info'>
-                                {release_time_avg
-                                    ? localize('{{- avg_release_time_in_minutes}} min', {
-                                          avg_release_time_in_minutes,
-                                      })
-                                    : '-'}
-                            </div>
-                        </Table.Cell>
-                        <Popover
-                            className='advertiser-page__popover-icon'
-                            alignment='top'
-                            message={localize(
-                                "These fields are based on the last 30 days' activity: Buy, Sell, Completion, and Avg. release time."
-                            )}
+                <div className='advertiser-page__stats--wrapper'>
+                    <Table>
+                        <ThemedScrollbars
+                            className='advertiser-page__horizontal-scroll'
+                            is_bypassed={!isMobile()}
+                            width='calc(100vw - 32px)'
+                            is_only_horizontal
+                            is_scrollbar_hidden
                         >
-                            <Icon icon='IcInfoOutline' size={16} />
-                        </Popover>
-                    </Table.Row>
-                </Table>
+                            <Table.Row className='advertiser-page__stats'>
+                                <Table.Cell className='advertiser-page__stats-cell'>
+                                    <div className='advertiser-page__stats-cell-header'>{localize('Total orders')}</div>
+                                    <div className='advertiser-page__stats-cell-info'>{total_orders_count || '-'}</div>
```

Comment:
Could we leverage `Text` component here instead of`info` class? Same for all other cases. As we're just using this class to apply text-specific styles.

Reply:
I didn't want to do this because I'd be repeating the same component with the same properties so I've created smaller components to handle this.